10.11/root/usr/share/container-scripts/mysql/README.md

```diff
@@ -0,0 +1,369 @@
+MariaDB 10.11 SQL Database Server Docker image
```

Comment:
This will be imo problem for more versions and images, but we should not use phrase "Docker image" and use "Container image" instead.
```suggestion
MariaDB 10.11 SQL Database Server Container image
```